mixer/pkg/lang/compiler/compiler.go

```diff
@@ -49,6 +49,12 @@ func New(finder ast.AttributeDescriptorFinder, functions map[string]ast.Function
 // CompileExpression creates a new parameterless IL function, using the given expression text as its body. Upon success,
 // it returns the id of the generated function.
 func (c *Compiler) CompileExpression(text string) (uint32, descriptor.ValueType, error) {
+	name := fmt.Sprintf("$expression%d", c.nextFnID)
```

Comment:
is a unique id required by CEL ?

Reply:
This is CEXL duplicate code removal. I deleted the duplicate `Compile` function down below (fixing the TODO there).